resources/views/components/accordion/mobile.blade.php

```diff
@@ -0,0 +1,24 @@
+{{--
+This mobile version only collapses on mobile, on desktop it's always open.
+
+## Example
+```
+<x-rapidez::accordion.mobile>
+    <x-slot:label>
+        Label
+    </x-slot:label>
+    <x-slot:content>
+        Content
+    </x-slot:content>
+</x-rapidez::accordion.mobile>
+```
+--}}
+
+<x-rapidez::accordion :$attributes>
```

Comment:
Maybe we should change the breakpoint from md to lg?

Reply:
for default rapidez where it is used in the footer md is used, if you want to use something else you can overwrite this file in your project